tests/benchmark.py

```diff
@@ -0,0 +1,44 @@
+# Testcase for benchmarking tsfresh feature extraction and selection
+import pytest
+import pandas as pd
+import numpy as np
+
+from tsfresh import extract_features, extract_relevant_features
+from tsfresh.feature_extraction.settings import ComprehensiveFCParameters, MinimalFCParameters
+
+
+def create_data(time_series_length, num_ids, random_seed=42):
+    np.random.seed(random_seed)
+
+    df = pd.concat([
+        pd.DataFrame({
+            "id": [i] * time_series_length,
+            "time": range(time_series_length),
+            "value": np.random.randn(time_series_length)
```

Comment:
I am not sure but it could be that some of our features actually profit from normally distributed values (e.g. he wavelet features). Would be interesting to use a real world dataset for this benchmarking.

Or as a short term solution, I would cumsum the values. Otherwise its not really a curve but just pure chaos